UKHomeOffice/sgmr-service
# SPCR-463 Handle repairOrModified = false to be a valid value

### AC / Description of changes
Bug scenario
- complete voyage report with customs question changedOrModified = **No**
- go to cya page
- click on 'change' for custom question
- click on submit

What should happen
- form submits

The Bug
- form was erroring with 'you must select answer for changed or modified'
This was because the value for changedOrModified is a boolean True or False in the JSON we get on the GET voyage call. False was being treated as a falsey value when in this scenario we need it to be truthy as `No / False` is a valid answer to the question

### To Test
- complete voyage report with customs question changedOrModified = **No**
- go to cya page
- click on 'change' for custom question
- click on submit
- > form should submit


### Notes

---

**Remember to merge to feature branches not `master`.**
